ckan/authz.py

```diff
@@ -119,6 +121,8 @@ def _get_user(username):
     except TypeError:
         # c is not available
         pass
+    except DetachedInstanceError:
```

Comment:
The stack trace wasn't particularly useful, but the Travis CI linter revealed this variable that was assigned but never used. I think I resolved this with this commit - https://github.com/neon-ninja/ckanext-oaipmh/commit/2a835955b45c558ff3872a7e19ddca56f76f2c52